addons/website_event_track/security/event_track_security.xml

```diff
@@ -14,7 +14,7 @@
 
     <record id="ir_rule_event_track_tag_public" model="ir.rule">
         <field name="name">Event Track Tag: public/portal: color = published</field>
-        <field name="model_id" ref="event.model_event_tag"/>
+        <field name="model_id" ref="website_event_track.model_event_track_tag"/>
```

Comment:
Strange diff ...

Reply:
That's why there is explanation about this strange diff in commit message :)

Reply:
Mmh, indeed ^^ shouldn't it be done in stable in a separate commit ?

Reply:
@tde-banana-odoo Ahh it's non updatable. May be we can adapt the migration script that fixes this one so the new record rule we added here does not conflict with this one.